backend/src/controllers/muralcollection.controller.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from "express";
+import { EmptyResultError } from "sequelize";
+import { MuralCollectionInterface } from "../models/muralcollection.model";
+import { MuralCollectionService } from "../services/muralcollection.service";
+
+export class MuralCollectionController {
+  public collectionService: MuralCollectionService = new MuralCollectionService();
+
+  // POST /collection
+  public async create(req: Request, res: Response) {
+    const collection: MuralCollectionInterface = req.body.tour;
```

Comment:
How come `.tour` is used here?

Reply:
omg, I was thinking about tour class when writing it, changing rn!